opg-elasticsearch-shared-data-1604/Dockerfile

```diff
@@ -7,18 +7,14 @@ ENV BEATS_DASHBOARD_VERSION=5.4.0
 #Needed to allow us to download from the es site
 RUN apk update && \
     apk add --no-cache ca-certificates wget openssl && \
-    update-ca-certificates
-
-RUN cd /tmp && \
-   wget https://artifacts.elastic.co/downloads/beats/beats-dashboards/beats-dashboards-${BEATS_DASHBOARD_VERSION}.zip && \
-   unzip beats-dashboards-${BEATS_DASHBOARD_VERSION}.zip
-
-RUN cd /tmp/beats-dashboards-${BEATS_DASHBOARD_VERSION} && \
-   cp -rv metricbeat/* /tmp/elasticsearchshareddata/beats && \
-   cp -rv filebeat/* /tmp/elasticsearchshareddata/beats
-
-RUN rm -rf /tmp/beats-dashboards-* && \
-   chmod -R 644 /tmp/elasticsearchshareddata
+    update-ca-certificates && \cd /tmp && \
+    wget https://artifacts.elastic.co/downloads/beats/beats-dashboards/beats-dashboards-${BEATS_DASHBOARD_VERSION}.zip && \
+    unzip beats-dashboards-${BEATS_DASHBOARD_VERSION}.zip && \
+    cd /tmp/beats-dashboards-${BEATS_DASHBOARD_VERSION} && \
+    cp -rv metricbeat/* /tmp/elasticsearchshareddata/beats && \
+    cp -rv filebeat/* /tmp/elasticsearchshareddata/beats && \
+    rm -rf /tmp/beats-dashboards-* && \
+    chmod -R 644 /tmp/elasticsearchshareddata
```

Comment:
We don't need the directories to be executable though, the python script reads the files and posts them to elasticsearch @tim-mcarthur-moj